spring-config/frontier-context.xml

```diff
@@ -63,19 +63,60 @@
 		<constructor-arg index="1"
 			value="#{systemEnvironment['MDB_PORT']}" />
 		<constructor-arg index="2" ref="serializerBean" />
+		<constructor-arg index="3" value="true" />
+	</bean>
+
+
+	<bean id="UriFilterBean"
+		class="org.dice_research.squirrel.data.uri.filter.UriFilterConfigurator">
+		<constructor-arg index="0" ref="mongoDBKnowUriFilter" />
+		<constructor-arg index="1">
+			<list>
+				<!-- <ref bean="depthFilter" /> -->
+			</list>
+		</constructor-arg>
+		<constructor-arg index="2" value="OR" />
+	</bean>
+
+	<!-- Triple Store sparql implementation -->
+	<bean id="uriRetrieverBean"
+		class="org.dice_research.squirrel.frontier.recrawling.SparqlBasedOutDatedUriRetriever" factory-method="create">
+		<constructor-arg index="0"
+			value="#{systemEnvironment['SPARQL_URL']}" />
+		<constructor-arg index="1"
+			value="#{systemEnvironment['SPARQL_HOST_USER']}" />
+		<constructor-arg index="2"
+			value="#{systemEnvironment['SPARQL_HOST_PASSWD']}" />
 	</bean>
+	
+	<!-- Dummy UriRetriever Implementation
+	<bean id="uriRetrieverBean"
+		class="org.dice_research.squirrel.frontier.recrawling.DummyUriRetriever">
+	</bean> -->
+	
+	
 
-	<bean id="knowUriFilterBean"
+
+	<bean id="mongoDBKnowUriFilter"
 		class="org.dice_research.squirrel.data.uri.filter.MongoDBKnowUriFilter">
 		<constructor-arg index="0"
 			value="#{systemEnvironment['MDB_HOST_NAME']}" />
 		<constructor-arg index="1"
 			value="#{systemEnvironment['MDB_PORT']}" />
+
 	</bean>
-	
+
+
+	<bean id="depthFilter"
+		class="org.dice_research.squirrel.data.uri.filter.DepthFilter">
+		<constructor-arg index="0" value="3" />
+	</bean>
```

Comment:
Yes, this means that this will have a maximum depth of 3. The depthFilter is not required and can be removed.
I wrote tutorial about configuring this, but waiting the deploy of this version to push it.